Library/Homebrew/rubocops/cask/url_legacy_comma_separators.rb

```diff
@@ -0,0 +1,38 @@
+# typed: true
+# frozen_string_literal: true
+
+require "forwardable"
+require "uri"
+
+module RuboCop
+  module Cop
+    module Cask
+      # This cop checks for version.before_comma and version.after_comma
+      class UrlLegacyCommaSeparators < Base
+        include OnUrlStanza
+        extend AutoCorrector
+
+        MSG_CSV = "Use 'version.csv.first' instead of 'version.before_comma' " \
+                  "and 'version.csv.second' instead of 'version.after_comma'."
+
+        def on_url_stanza(stanza)
+          return if stanza.stanza_node.type == :block
+
+          url_node = stanza.stanza_node.first_argument
+
+          legacy_comma_separator_pattern = /version.(before|after)_comma/
```

Comment:
```suggestion
          legacy_comma_separator_pattern = /version\.(before|after)_comma/
```

Another thought (following on from Slack): Do you need to escape this `.` here to avoid it matching any character, eg `versionnbefore_comma` or some such?